src/dmd/backend/cv8.d

```diff
@@ -514,23 +559,23 @@ uint cv8_addfile(const char *filename)
      * Unlike C, there won't be lots of .h source files to be accounted for.
      */
 
-    uint length = F3_buf->size();
-    ubyte *p = F3_buf->buf;
+    uint length = cast(uint)F3_buf.size();
+    ubyte *p = F3_buf.buf;
     size_t len = strlen(filename);
 
     // ensure the filename is absolute to help the debugger to find the source
     // without having to know the working directory during compilation
-    static char cwd[260];
-    static uint cwdlen;
+    __gshared char[256] cwd;
```

Comment:
Why change the size here? MAX_PATH is defined as 260.

Reply:
A stupid mistake of mine.